pilot/const/function_calls.py

```diff
@@ -431,75 +450,6 @@ def command_definition(description_command='A single command that needs to be ex
     }
 }
 
-GET_FILES = {
```

Comment:
These few schema definitions were used only in the now-removed functionality atlhough naming suggests they were more generic.